src/app/app-routing.module.ts

```diff
@@ -1,9 +1,14 @@
 import { NgModule } from '@angular/core';
-import {RouterModule, Routes} from "@angular/router";
+import { RouterModule, Routes } from "@angular/router";
+import { LandingComponent } from './landing/landing/landing.component';
+import { ProfileComponent } from './profile/profile/profile.component';
 
 const routes: Routes = [
-  { path: '', redirectTo: 'properties', pathMatch: 'full' },
+  { path: '', redirectTo: 'home', pathMatch: 'full' },
+  { path: 'home', component: LandingComponent },
+  { path: 'profile', component: ProfileComponent },
   { path: 'properties', loadChildren: () => import('./properties/properties.module').then(m => m.PropertiesModule) },
+  { path: 'profile', component: ProfileComponent}
```

Comment:
There is already a route defined for the 'profile' path, this should be removed.